pilot/pkg/config/file/store.go

```diff
@@ -282,7 +294,7 @@ func (s *KubeSource) parseContent(r *collection.Schemas, name, yamlText string)
 		if err != nil {
 			e := fmt.Errorf("error reading documents in %s[%d]: %v", name, chunkCount, err)
 			scope.Warnf("%v - skipping", e)
-			scope.Debugf("Failed to parse yamlText chunk: %v", yamlText)
+			scope.Debugf("Failed to parse yamlText chunk")
```

Comment:
If yamlText is not needed, is it better to use `scope.Debug` instead of `scope.Debugf`?